openstack/sharedfilesystems/v2/sharetypes/results.go

```diff
@@ -68,3 +68,28 @@ func ExtractShareTypes(r pagination.Page) ([]ShareType, error) {
 type GetDefaultResult struct {
 	commonResult
 }
+
+// ExtraSpecs contains all the information associated with extra specifications
+// for an Openstack ShareType.
+type ExtraSpecs struct {
```

Comment:
If `ExtraSpecs` is just a map with [varying] key-value pairs, we don't need to wrap it in a `struct`:
```go
type ExtraSpecs map[string]interface{}
```